packages/backend/src/assets/ai.json

```diff
@@ -21,6 +21,26 @@
       "languages": ["python"],
       "frameworks": ["streamlit", "langchain"]
     },
+    {
+      "id": "chatbot2",
```

Comment:
Why adding this recipe?

Reply:
I think it is just preliminary change to test PR with a trigger.

Reply:
to test triggering the workflow, it will be removed

Reply:
removed, to test the pipeline re-trigger https://github.com/containers/podman-desktop-extension-ai-lab/actions/runs/13267827590/job/37039679174?pr=2505 after merge